blocks/video-player-block/features/video-player/default.jsx

```diff
@@ -94,7 +96,9 @@ const VideoPlayer = (props) => {
     <div className="container-fluid video-promo">
       <div className="row">
         <div className="col-sm-xl-12">
-          {alertBadge && <AlertBadge>{alertBadge}</AlertBadge>}
+          <div className="padding-sm-bottom">
```

Comment:
@amyerz Do we want this div to appear even if there is no alertBadge?

Reply:
updated!